README.md

```diff
@@ -17,3 +17,25 @@ Install/update your vendors:
 $ curl http://getcomposer.org/installer | php
 $ php composer.phar install
 ```
+
+## Configuration and Execution
+
+1. Installing dependencies via composer
+2. Setting up a web-server, with PHP support, pointing to **./Fixtures**
+3. In case the **Fixture** directory is not reachable via http://localhost change in **behat.yml**
+ * `base_url` for `MinkExtension` 
+ * `base_url` for `BehatExtension`
+ ```yaml
+ Behat\MinkExtension:
+       default_session: selenium2
+       javascript_session: selenium2
+       base_url:  'http://localhost'
+ 
+       selenium2: ~
```

Comment:
I always wanted to know what `~` means here. Is it Behat specific or Yaml specific stuff.